src/compressed_tensors/compressors/model_compressor.py

```diff
@@ -146,16 +159,29 @@ def from_pretrained_model(
     def parse_sparsity_config(compression_config: Dict) -> Union[Dict, None]:
         if compression_config is None:
             return None
+        if SPARSITY_CONFIG_NAME not in compression_config:
+            return None
+        if hasattr(compression_config, SPARSITY_CONFIG_NAME):
+            # for loaded HFQuantizer config
+            return getattr(compression_config, SPARSITY_CONFIG_NAME)
         return compression_config.get(SPARSITY_CONFIG_NAME, None)
```

Comment:
Why are both these cases needed??

Reply:
if this is an HF CompressedTensors quantization config, then it won't be a dict so we can't access it with the dict getters